test/functional/test_runner.py

```diff
@@ -495,14 +494,7 @@ def get_next(self):
                         status = "Failed"
                     self.num_running -= 1
                     self.jobs.remove(job)
-                    if self.logging_level == logging.DEBUG:
-                        clearline = '\r' + (' ' * dot_count) + '\r'
-                        print(clearline, end='', flush=True)
-                        dot_count = 0
                     return TestResult(name, status, int(time.time() - start_time)), testdir, stdout, stderr
-            if self.logging_level == logging.DEBUG:
-                print('.', end='', flush=True)
-                dot_count += 1
```

Comment:
I think it was nice to have some indicator that something was running. I'd prefer to keep it.

Reply:
I agree: running functional tests, without any progress indicator, it's not a good incentive to run functional tests.